### ISSUE TITLE
When navigating with the arrows by a combo box in abby fine reader 12 the nvda does not announce the currently focused option

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8615]
### Steps to reproduce:
open abby fine reader 12, I believe it might be an evaluation license.
You also need to have a scanner connected to your computer.
If the window with shortcuts is opened, close esc.
Press alt to open files, with the arrow until
to scan pages.
walk tab to the resolution option.
Now walk with arrows up and down.
The options are changed in the combo box.

### Actual behavior:
the nvda is mute.

### Expected behavior:
as the options are changed the nvda should announce the new values that gain focus.

#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 15781
#### Windows version:
10 17134.191
#### Name and version of other software in use when reproducing the issue:
abby fine reader 12
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
log:
IO - inputCore.InputManager.executeGesture (17:25:42.489):
Input: kb(laptop):tab
IO - speech.speak (17:25:42.569):
Speaking [u'Resolu\xe7\xe3o:  caixa de combina\xe7\xe3o  1200  dpi  Alt+\xe7']
IO - inputCore.InputManager.executeGesture (17:25:50.075):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (17:25:52.836):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (17:25:56.670):
Input: kb(laptop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (17:25:56.795):
Developer info for navigator object:
name: u'Resolu\xe7\xe3o:'
role: ROLE_COMBOBOX
states: STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.IAccessible object at 0x050EC5F0>
Python class mro: (<class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=1102, top=497, width=252, height=24)
value: u'300  dpi (para texto normal)'
appModule: <'appModuleHandler' (appName u'finereader', process ID 14216) at address 4fdee70>
appModule.productName: u'ABBYY FineReader'
appModule.productVersion: u'12.0.101.483'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 2033172
windowClassName: u'AWL:2EE50000:80:0:0:0:0'
windowControlID: 5087
windowStyle: 1375797248
windowThreadID: 6856
windowText: u'300  dpi (para texto normal)'
displayText: u'300  dpi (para texto normal)'
IAccessibleObject: <POINTER(IAccessible) ptr=0x7a12340 at 50a07b0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=2033172, objectID=-4, childID=0
IAccessible accName: u'Resolu\xe7\xe3o:'
IAccessible accRole: ROLE_SYSTEM_COMBOBOX
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: u'300  dpi (para texto normal)'

**Comment by leonardder** [comment]
1. What happens if you open the combo box with alt+down arrow?
2. Can you reproduce this with Narrator or another screen reader?

**Comment by fernando-jose-silva** [comment]
by pressing alt + arrows below the nvda still does not read.

the jaws 2018 will usually these boxes.

 

De: Leonard de Ruijter <notifications@github.com> 
Enviada em: sábado, 11 de agosto de 2018 14:32
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] When navigating with the arrows by a combo box in abby fine reader 12 the nvda does not announce the currently focused option (#8615)

 

1.	What happens if you open the combo box with alt+down arrow?
2.	Can you reproduce this with Narrator or another screen reader?

**Comment by Adriani90** [comment]
@fernando-jose-silva the app design in ABBYY fine reader changed alot. Can you also reproduce this in ABBYY finereader 14 or 15? I cannot reproduce it. Now those combo boxses are designed as button submenus and the states are reported correctly.

**Comment by fernando-jose-silva** [comment]
Using abby fine reader 14 I found a combo box.
In the program's initial screen choose settings, in the settings window choose ocr.
In the first tab you will find the combo box:
PDF Recognition Mode
And unfortunately I was able to reproduce the problem, nvda not reading while walking through the options with arrows, even if the options are changed which can be seen by taking focus off the combo box.

log:
[reader14.txt](https://github.com/nvaccess/nvda/files/4062519/reader14.txt)
